src/main/resources/application.yml

```diff
@@ -13,7 +13,17 @@ server:
 oauth:
   clientId: ${CLIENT_ID:9fbd4ae2-2db3-44c7-9544-88e80255b56e}
   clientSecret: ${CLIENT_SECRET:test}
-  checkTokenEndpointUrl: ${CHECK_TOKEN_URL:http://localhost:8080/oauth/check_token}
+  serviceUrl: ${OAUTH_SERVICE_URL:http://localhost:8080}
+  checkTokenEndpointUrl: "${oauth.serviceUrl}/oauth/check_token"
+  tokenUrl: "${oauth.serviceUrl}/oauth/token"
+
+identity:
+  identityAPIUrl: "${oauth.serviceUrl}/api/identities"
+
+govNotify:
+  key: ${GOV_NOTIFY_API_KEY:key}
+  template:
+    lineManager: ${GOV_NOTIFY_LINEMANAGER_TEMPLATE_ID:template}
```

Comment:
DOCUMENTATION - missing and terraform values also

Reply:
Only documentation I'm aware of is accounts.text in keybase. Will add a line in there for Line manager id.

Reply:
Have zero knowledge of terraform. @robertmarks is helping out.

Reply:
`README.md` is our documentation

Reply:
Added needed env to README.md table